APE0.rst

```diff
@@ -0,0 +1,385 @@
+  formal capacity,
+* Act as the decision-making body where all other methods for achieving
+  consensus have failed,
+* Operate openly and provide transparency and opportunity for community input
+  into its activities.
+* Act as a point of contact with other organizations.
+
+Responsibilities and Authority
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The Coordination Committee has broad authority to make decisions about the
+Project. For example, they can:
+
+* Accept or reject APEs and updates to APEs (except this one - see below);
+* Enforce or update the Project's code of conduct
+* Manage any project assets (including financial and physical assets)
+* Delegate parts of their authority to other subcommittees or processes
+
+However, they cannot modify the Governance of the Project (i.e., this document),
+except during the initial adoption of this Charter as specified herein.
+
+While the Coordination Committee has broad authority, it has the responsibility
+to work towards consensus whenever practical. The Coordination Committee
+should look for ways to use its authority as little as possible, striving 
+to involve the community in decisions whenever possible, and delegating
+responsibilities.
+
+When delegating its authority to create roles or make appointments, the
+Coordination Committee must solicit community feedback for no less than two
+weeks.
+
+Making Decisions
+^^^^^^^^^^^^^^^^
+To use its authority, the Coordination Committee must first attempt to reach
+a consensus. If a consensus cannot be reached the Committee votes. Every
+Coordination Committee member must either vote or explicitly abstain. Members
+with a conflict of interest on a particular vote must abstain. The final
+decision regarding what constitutes a conflict of interest is determined by the
+Ombudsperson in accordance with the conflict of interest policy. Passing
+decisions require a strict majority of non-abstaining Coordination Committee
+members.
+
+Whenever possible, the Coordination Committee's deliberations shall be held in
+public in a manner that permits discussion and comments from the community.
+Final decisions made by the Coordination Committee will be published in a
+publicly visible location.
+
+Electing the Coordination Committee
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A Coordination Committee election is held annually to fill any vacant seats.
+Notice of the election must be posted via the Project’s standard communication
+channels (reaching all active members) at least one month before the election begins.
+The election consists of two phases, each lasting two weeks:
+
+* Phase 1, Nominations: Candidates will be nominated and confirm their interest
+  in serving. Candidates can be nominated by anyone, including themselves, and
+  do not have to be Voting Members to be nominated. Candidates should provide
+  a brief statement in support of their candidacy.
+* Phase 2, Voting: Each Voting Member can vote for zero or more of the
```

Comment:
Minor, but could `Voting Member` be `[Voting Member](#Voting_Member)` (or whatever is syntactically correct) so it links to the definition below?  I've been reading linearly, and I arrived here before learning what a voting member is.  I wasn't clear how anonymous voting could be managed before scrolling down further.

Reply:
`Voting Member <#Voting_Member>`_, maybe?  I'm not sure exactly how this will get rendered.

Reply:
Good idea!  Was a bit more complex than I thought, but done now.